mne/io/tests/test_pick.py

```diff
@@ -244,6 +244,32 @@ def test_pick_seeg_ecog():
     assert_equal(len(pick_types(raw.info, meg=False, seeg=True, ecog=True)), 0)
 
 
+def test_pick_dbs():
+    """Test picking with DBS."""
+    # gh-8739
+    names = 'A1 A2 Fz O OTp1 OTp2 OTp3'.split()
+    types = 'mag mag eeg eeg dbs dbs dbs'.split()
+    info = create_info(names, 1024., types)
+    picks_by_type = [('mag', [0, 1]), ('eeg', [2, 3]), ('dbs', [4, 5, 6])]
+    assert_indexing(info, picks_by_type)
+    assert_array_equal(pick_types(info, meg=False, dbs=True), [4, 5, 6])
+    for i, t in enumerate(types):
+        assert_equal(channel_type(info, i), types[i])
+    raw = RawArray(np.zeros((len(names), 7)), info)
+    events = np.array([[1, 0, 0], [2, 0, 0]])
+    epochs = Epochs(raw, events=events, event_id={'event': 0},
+                    tmin=-1e-5, tmax=1e-5,
+                    baseline=(0, 0))  # only one sample
+    evoked = epochs.average(pick_types(epochs.info, meg=True, dbs=True))
+    e_dbs = evoked.copy().pick_types(meg=False, dbs=True)
+    for lt, rt in zip(e_dbs.ch_names, [names[4], names[5], names[6]]):
+        assert lt == rt
+    # Deal with constant debacle
```

Comment:
Can you elaborate in this comment? What was the challenge? I didn't see any `803` in `constants.py` in `master` so I'm not sure...

Reply:
Actually, I had copied this line from elsewhere and I forgot to remove the comment. We can ignore it, I removed the comment now. I should probably get better at reviewing my own code before committing

Reply:
Pro tip: `git add -p name_if_file.py` adds files in patch mode, forcing you to look at the changes in the file before staging them. I find it helps me avoid accidentally committing cruft, and also nudges me toward grouping changes semantically

Reply:
That's great advice thank you so much :)